src/sentry/static/sentry/app/views/organizationDiscover/result/utils.jsx

```diff
@@ -231,3 +232,37 @@ const LightGray = styled.span`
 const DarkGray = styled.span`
   color: ${p => p.theme.gray5};
 `;
+
+/**
+ * Downloads a Snuba result object as CSV format
+ *
+ * @param {Object} result Result received from Snuba
+ * @param {Object} result.data Result data object from Snuba
+ * @param {String} result.meta Result metadata from Snuba
+ * @returns {Void}
+ */
+export function downloadAsCsv(result) {
+  const {meta, data} = result;
+  const headings = meta.map(({name}) => name);
+
+  const csvContent = Papa.unparse({
+    fields: headings,
+    data: data.map(row => {
+      return headings.map(col => quoteUnsafeStrings(row[col]));
```

Comment:
Is `papaparse` able to handle this?

Reply:
I don't think so, I was hoping to not have to do this manually too

Reply:
What if we just quoted every field (I'm assuming papaparse would then handle escaping `"`)?

```
Set quotes to true to always enclose each field in quotes, or an array of true/false values correlating to specific to columns to force-quote. The character used to quote can be customized using quoteChar. The character used to escape the quoteChar within a field can be customized using escapeChar. The delimiter can be any valid delimiting character. The newline character(s) may also be customized. Setting header to false will omit the header row.
```

Reply:
I wouldn't want to get a CSV like that